test/popupview_form_test.js

```
@@ -0,0 +1,236 @@
+      console.log(`Event listener added: ${event}`);
+    }
+  },
+  utils: {
+    makeAvatarUrl: (params) => {
+      return params.avatarUrl || '';
+    }
+  }
+};
+
+// Execute the PopupViewForm code
+eval(popupViewFormCode);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)